src/windows-emulator/devices/afd_endpoint.cpp

```diff
@@ -547,6 +624,286 @@ namespace
                 return STATUS_ADDRESS_ALREADY_ASSOCIATED;
             }
 
+            if (c.io_status_block)
+            {
+                IO_STATUS_BLOCK<EmulatorTraits<Emu64>> block{};
+                block.Status = STATUS_SUCCESS;
+                block.Information = 0;
+                c.io_status_block.write(block);
+            }
```

Comment:
i don't think this is needed. this is automatically done here:
https://github.com/momo5502/sogen/blob/main/src/windows-emulator/io_device.hpp#L115